src/components/header/components/Nav.tsx

```diff
@@ -1,39 +1,22 @@
-import React, { HTMLProps, useContext } from 'react';
-import classNames from 'classnames';
-import { Container } from '../../layout';
-import { Close as CloseIcon } from '../../icons';
-import HeaderContext, { IHeaderContext } from '../HeaderContext';
+import React, { HTMLProps } from 'react';
```

Comment:
These changes were undone based on:

>I'm a fan of adding a separate implementation of Header.Nav rather than replacing it, as pretty much nearly every service currently using the package is using it.

From #56 

@Tomdango if you get a chance I think this PR is ready for review